### PR TITLE
Task_DEV-1

### PR DESCRIPTION
**Pull request by OlgaYurkova**

### CODE REVIEW

File: DEV-1/SearchSeguence.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline DEV-1/SearchSeguence.cs]: Not used, remove

File: DEV-1/SearchSeguence.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    /// <summary>
+    ///The SequenceSearch class is the main class of the program that works with the string
+    ///It finds the maximum number of identical consecutive characters in the string
+    /// </summary>
+
+    class SearchSequence
+    {
+        static void Main(string[] args)
+        {
+
```
Comment (IgorKheidorov) [inline DEV-1/SearchSeguence.cs:15]: Code style violation, empty string

File: DEV-1/SearchSeguence.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            if (args.Length>=1 && args.Length < 3)
```
Comment (IgorKheidorov) [inline DEV-1/SearchSeguence.cs]: Two arguments are allowed? I am sure that args.Length == 1

File: DEV-1/SearchSeguence.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            {
+                string inputString = args[0];
+                int counterOfChars = 1;
+                int maxSequence = 0;
+                for (int i = 0; i < inputString.Length-1 ; i++)
```
Comment (IgorKheidorov) [inline DEV-1/SearchSeguence.cs]: Error, input only one symbol, result is 0, but has to be 1

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

**Review by AlexanderSoroka — APPROVED** [review]
Not accepted as some comments were just ignored

**Pull request merged**